### PR TITLE
added lab1 main functionality

### PR DESCRIPTION
**Pull request by BohdanBodakva**

### CODE REVIEW

**Review by DenysDoskochynskyi — CHANGES_REQUESTED** [review]
Please add linter rules from my repo:)

**Comment by BohdanBodakva** [comment]
Added linter and fixed all issues found by it. Now it all works well.

File: lib/main.dart [hunk 1]
```diff
@@ -10,116 +10,147 @@ class MyApp extends StatelessWidget {
...
-          // how it positions its children. Here we use mainAxisAlignment to
-          // center the children vertically; the main axis here is the vertical
-          // axis because Columns are vertical (the cross axis would be
-          // horizontal).
-          //
-          // TRY THIS: Invoke "debug painting" (choose the "Toggle Debug Paint"
-          // action in the IDE, or press "p" in the console), to see the
-          // wireframe for each widget.
           mainAxisAlignment: MainAxisAlignment.center,
           children: <Widget>[
```
Comment (DenysDoskochynskyi) [inline lib/main.dart]: You can remove that <Widget> it's not necessary

File: lib/main.dart [hunk 2]
```diff
@@ -10,116 +10,147 @@ class MyApp extends StatelessWidget {
   // This widget is the root of your application.
```
Comment (DenysDoskochynskyi) [inline lib/main.dart]: remove comments)

File: lib/main.dart [hunk 3]
```diff
@@ -10,116 +10,147 @@ class MyApp extends StatelessWidget {
...
-        //
-        // Notice that the counter didn't reset back to zero; the application
-        // state is not lost during the reload. To reset the state, use hot
-        // restart instead.
-        //
-        // This works for code too, not just values: Most code changes can be
-        // tested with just a hot reload.
-        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
+        colorScheme: ColorScheme.fromSeed(seedColor: Colors.green),
         useMaterial3: true,
```
Comment (DenysDoskochynskyi) [inline lib/main.dart]: No need to set that value(be default it's true)

**Review by DenysDoskochynskyi — APPROVED** [review]
Just some minor comments, but everything is nice)

File: lib/main.dart (hunk 1 above)
Comment (BohdanBodakva) [reply lib/main.dart]: Fixed, thanks.

File: lib/main.dart (hunk 2 above)
Comment (BohdanBodakva) [reply lib/main.dart]: Fixed, thanks.

File: lib/main.dart (hunk 3 above)
Comment (BohdanBodakva) [reply lib/main.dart]: Fixed, thanks.

**Pull request merged**